code-challenges.js

```diff
@@ -31,24 +66,83 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+
+
+
+
+
+
+
+
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
 
+//I am gonna see if I can somehow use a accessor for this.
+// First I need to create a variable with 'var'.
+// Next for the variable I need a name, probably 'padresWorldSeriesRuns'.
+// Then stick both const together so I can just call upon my var
+// Lastly all I should need is .length property 
+
+
+
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+var padresWorldSeriesRuns = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+console.log(padresWorldSeriesRuns.length)
+
+//Found concat is exactly what I needed
+//My theory worked! Easy peasy
+
+
+
+
+
+
+
+
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
 
+// When I saw reverse, I instantly thought of some array mutator or accessor must do the trick.
+// I will have to do var again for variable delcaration.
+// I am going to need something like reverse (did not work).
+
 const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
+ 
+const cohortReversed = currentCohort
+    .split(``)
+    .reverse()
+    .join(``)
+
+console.log(cohortReversed)
+
+// Had to look this up, needed a combination mutators and accessors. Looks easy now but its kinda insane.
```

Comment:
For sure, reverse only works on arrays which makes this problem a little more tricky than at first glance.